src/gridtools/clang/verify.hpp

```diff
@@ -133,6 +133,28 @@ class verifier {
 
     return verified;
   }
+  template <class StorageType>
+  void printStorage(StorageType storage) {
+    using namespace gridtools;
+
+    storage.sync();
+
+    auto storage_v = make_host_view(storage);
```

Comment:
can you create a read only view? (since otherwise we need to sync at the end of the method)